### PR TITLE
Whois Integration

### PR DESCRIPTION
**Pull request by amshamah419**
## Status
Ready

## Related Issues
fixes: demisto/etc#16414

## Description
Moved package inside integration to allow more rapid deployment of regex changes.

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [x] Documentation (with link to it) https://github.com/demisto/etc/issues/17130
- [x] Code Review

### CODE REVIEW

**Comment by amshamah419** [comment]
I originally was going to make this a V2, but there was a decision made to keep the current integration and improve it.

File: Integrations/Whois/Whois.py [hunk 1]
```diff
@@ -0,0 +1,8289 @@
...
+    },
+    "hotmail": {
+        "_type": "newgtld",
+        "adapter": "none"
+    },
+    "house": {
+        "_group": "donuts",
+        "_type": "newgtld",
+        "host": "whois.nic.house"
+
```
Comment (yuvalbenshalom) [inline Integrations/Whois/Whois.py]: get the domain argument here

Comment (yuvalbenshalom) [inline Integrations/Whois/Whois.py]: catch value error when using `strftime`
this is relevant also to the following lines.

Comment (yuvalbenshalom) [inline Integrations/Whois/Whois.py]: why do you expect OSError here?

Comment (yuvalbenshalom) [inline Integrations/Whois/Whois.py]: consider moving the code down after the `grammar` so you will not have a short code "hiding" between 2 huge dictionary initialisation

File: Integrations/Whois/Whois.yml [hunk 2]
```diff
@@ -191,6 +68,84 @@ script:
     - contextPath: Whois.Raw
       description: Raw Output
       type: string
+    - contextPath: Whois.Administrator.country
```
[the remark this answers is not in the document]
Comment (amshamah419) [reply Integrations/Whois/Whois.yml]: I was originally dumping the results into the context for the Admin, Tech Admin, Registrant, and Billing keys. I changed it to use the createContext keyTransformer to capitalise these keys.

**Comment by glicht** [comment]
How is this related to #3567 ? which one are we merging in the end of day?

**Comment by amshamah419** [comment]
@glicht This is the pull request that will be in master at the end of the day. We had to split it into two pull requests since the original was a single page integration. After `whois-pkg` is merged, then this one will be merged so we can track the changes.

**Pull request merged**